src/ITensorNetworks.jl

```diff
@@ -53,7 +53,8 @@ using NamedGraphs:
   parent_graph,
   vertex_to_parent_vertex,
   parent_vertices_to_vertices,
-  not_implemented
+  not_implemented,
+  parent
```

Comment:
Is this needed? Isn't it just an overload of `Base.parent`, and therefore is already available in the namespace?

Reply:
You resolved but I'm not sure how it is resolved.

Reply:
Good spot, I will remove from imports. Sorry about that

Reply:
No problem, thanks.